registry/deploying.md

```diff
@@ -1,94 +1,240 @@
 ---
 description: Explains how to deploy a registry
 keywords: registry, on-prem, images, tags, repository, distribution, deployment
-title: Deploying a registry server
+title: Deploy a registry server
 ---
 
-You need to [install Docker version 1.6.0 or newer](/engine/installation/index.md).
+Before you can deploy a registry, you need to install Docker on the host.
+A registry is an instance of the `registry` image, and runs within Docker.
 
-## Running on localhost
+This topic provides basic information about deploying and configuring a
+registry. For an exhaustive list of configuration options, see the
+[configuration reference](configuration.md).
 
-Start your registry:
+If you have an air-gapped datacenter, see
+[Considerations for air-gapped registries](#considerations-for-air-gapped-registries).
 
-    docker run -d -p 5000:5000 --restart=always --name registry registry:2
+## Run a local registry
+
+Use a command like the following to start the registry container:
```

Comment:
We should point out that the example here is just for testing / trying out, because;

- it's insecure (there's no TLS, nor authentication in place, and the port is published, so anyone has access to your registry)
- there's no volume used, so images are stored on the container's filesystem; removing the container removes all your images